### PR TITLE
Add nilaway - nil checking 

### PR DESCRIPTION
**Pull request by taimoorgit**
## Issues

https://github.com/OpsLevel/team-platform/issues/186

## Changelog

- [x] Add nil checks to cache.go mandated by `nilaway`
- [ ] ~~Make a `changie` entry~~

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -21,8 +21,8 @@ tasks:
     desc: Formatting and linting
     cmds:
       - test -z "$(gofumpt -d -e . | tee /dev/stderr)"
-      - go vet ./...
```
Comment (taimoorgit) [inline Taskfile.yml:24]: golangci-lint run already includes go vet, so I removed this line.

**Comment by davidbloss** [comment]
Great find! 🎊 

As part of `task fix` can you add
`- nilaway -fix -test=false -include-pkgs='github.com/opslevel/opslevel-go' ./...` ?

**Comment by taimoorgit** [comment]
> Great find! 🎊
> 
> As part of `task fix` can you add `- nilaway -fix -test=false -include-pkgs='github.com/opslevel/opslevel-go' ./...` ?

@davidbloss Added!

**Review by davidbloss — APPROVED** [review]
LGTM 🎉

**Pull request merged**